freebridgefe/src/assets/styles/index.css

```
@@ -35,4 +45,177 @@
+        linear-gradient(180deg, #fbfdff 0%, #ffffff 22%, #fcfdff 100%);
+    color: var(--fb-text);
+    font-family: Inter, Pretendard, Roboto, system-ui, -apple-system, BlinkMacSystemFont, "Segoe UI", sans-serif;
+}
+
+@layer components {
+    .fb-page-shell {
+        @apply min-h-screen;
+        background:
+            radial-gradient(circle at top left, rgba(33, 175, 191, 0.04), transparent 18%),
+            radial-gradient(circle at top right, rgba(0, 212, 218, 0.05), transparent 15%),
+            linear-gradient(180deg, #fbfdff 0%, #ffffff 20%, #fcfdff 100%);
```

Stylelint (17.4.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)